content/pt/community/end-user/issue-participation.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Vote e Comente em Issues do GitHub
+linkTitle: Participação em Issues
+description: >-
+  Nos ajude a priorizar o trabalho no OpenTelemetry contribuindo para issues do
+  GitHub com seus votos e comentários.
+weight: 5
+cSpell:ignore: triagers upvote
+default_lang_commit: 653100b19c63cd3db6e19e578455d16c9d5949f8
+---
+
+No OpenTelemetry, sempre estamos procurando maneiras de facilitar para que
+nossos usuários finais e membros da comunidade tenham suas vozes ouvidas.
+Sabemos que nem todos podem participar de reuniões de Grupo de Interesse Especial (SIG) ou participar de todas
```

Comment:
```suggestion
Sabemos que nem todos podem participar de reuniões de Grupo de Interesse Especial (SIG, Special Interest Group) ou participar de todas
```

Reply:
Perhaps you could add something like the following in the prompt:

"When using acronyms, always show also the original term, such as: 'Grupo de Interesse Especial (SIG, Special Interest Group)' "